lib/internal/Magento/Framework/Console/Cli.php

```diff
@@ -256,4 +254,23 @@ private function documentRootResolver(array $config = []): array
 
         return array_merge_recursive($config, $params);
     }
+
+    /**
+     * Generate and return the Command Loader
+     *
+     * @throws \LogicException
+     * @throws \BadMethodCallException
+     */
+    private function getCommandLoader(): Console\CommandLoader\CommandLoaderInterface
+    {
+        $commandLoaders = [];
+        if (class_exists(SetupCommandLoader::class)) {
+            $commandLoaders[] = new SetupCommandLoader($this->serviceManager);
+        }
+        $commandLoaders[] = $this->objectManager->create(CommandLoader::class);
```

Comment:
Can we define a list of these loaders via di.xml, so custom modules could also define their own loaders if needed?

Reply:
It was done in the CommandLoader.